vpr/src/route/route_common.cpp

```diff
@@ -1147,27 +1147,27 @@ t_bb load_net_route_bb(ClusterNetId net_id, int bb_factor) {
     const t_rr_node& source_node = device_ctx.rr_nodes[driver_rr];
     VTR_ASSERT(rr_graph.node_type(RRNodeId(driver_rr)) == SOURCE);
 
-    VTR_ASSERT(source_node.xlow() <= source_node.xhigh());
-    VTR_ASSERT(source_node.ylow() <= source_node.yhigh());
+    VTR_ASSERT(rr_graph.node_xlow(source_node.id()) <= rr_graph.node_xhigh(source_node.id()));
```

Comment:
Should we use ``RRNodeId(driver_rr)`` here because it will save our effort later down the road.
The data type in ``net_rr_terminals`` will adopt ``RRNodeId``.